Directory.Build.props

```diff
@@ -1,10 +1,14 @@
 <Project>
   <PropertyGroup>
-    <PackageIconUrl>https://pbs.twimg.com/profile_images/487954549441691649/O3KsHAsb_400x400.png</PackageIconUrl>
+    <PackageIcon>icon.png</PackageIcon>
```

Comment:
Lets use PascalCase for filenames as all the other files in repo are that way. Also maybe move it in MonoGame.Framework directory?

Reply:
Tools use this as well.